plugins/database/src/low_level_odbc.cpp

```diff
@@ -502,6 +500,9 @@ _cllExecSqlNoResult(
     if ( stat == SQL_SUCCESS ||
             stat == SQL_SUCCESS_WITH_INFO ||
             stat == SQL_NO_DATA_FOUND ) {
+        auto ec = SQLRowCount(myHstmt, (SQL_INT_OR_LEN*) &rowCount); // NOLINT(google-readability-casting)
```

Comment:
Is `ec` ever checked after this call?

Reply:
Nope, it's only used in the logging statement in the next line (same as the original code-- I made no deletions here). If that's no longer necessary, it can be removed.

Reply:
Got it. Please mark it `const` since it's not modified.